learners/setup.md

```diff
@@ -2,53 +2,122 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+## Download Data
 
-## Data Sets
+During this lessons we will be using two microscopy images and creating images
+of our own. Create a working directory, you can name it anything you like but
+we recommend `microsopy-ideas`.
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+Download
+ [00001_01.ome.tiff](https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/00001_01.ome.tiff)
+ from the open microscopy environment pages to your working directory.
+
+Download
+[Plate1-Blue-A-12-Scene-3-P3-F2-03.czi](https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/Plate1-Blue-A_TS-Stinger/Plate1-Blue-A-12-Scene-3-P3-F2-03.czi)
+from the open microscopy environment pages to your working dircetory.
 
 ## Software Setup
 
-::::::::::::::::::::::::::::::::::::::: discussion
+During these lessons we will use the [napari](https://napari.org/stable/) image
+ viewer and the
+[napari-aicsimagio](https://github.com/AllenCellModeling/napari-aicsimageio)
+ plugin to load different image types.
+For beginners we recommend using Anaconda to install and manage these
+packages. The first step is to
+[download and install Anaconda](https://www.anaconda.com/download#downloads).
+
+The remaining instructions are written assuming you will be using the
+Anaconda prompt. If you are unfamiliar with Anaconda it may be wise to
+review the
+ [Anaconda getting started pages](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html).
+
+Start the Anaconda prompt and run the following commands (lines starting
```

Comment:
Also, I wouldn't use 'anaconda prompt' here. Technically, the anaconda prompt is only used on windows, mac/linux just use their standard terminal

Reply:
39c743c